tools/hardware/config.py

```diff
@@ -29,6 +31,12 @@ def get_device_page_bits(self) -> int:
         ''' Get page size in bits for mapping devices for this arch '''
         return self.get_page_bits()
 
+    def align_memory(self, regions: Set[Region]) -> List[Region]:
+        ''' Given a set of regions, sort them and align the first so that the ELF loader will be able to load the kernel into it. Will return the aligned
```

Comment:
Break the line to improve readbility:
```suggestion
        ''' Given a set of regions, sort them and align the first so that the 
        ELF loader will be able to load the kernel into it. Will return the
        aligned memory region list, a set of any regions of memory that were
        aligned out and the physBase value that the kernel will use. '''
```